2 - RTB/Documentazione interna/Verbali/23-12-03/23-12-03.typ

```diff
@@ -0,0 +1,96 @@
+#import "/template.typ":*
+
+#show: project.with(
+  date: "03/12/23",
+  docType: "verbale",
+  authors: (
+    "Gardin Giovanni",
+  ),
+  reviewers: (
+    "Todesco Mattia",
+  ),
+  missingMembers: (
+    "Carraro Riccardo", "Zaccone Rosario",
+  ),
+  timeStart: "15:00",
+  timeEnd: "17:00",
+);
+
+= Ordine del giorno
+- Rotazione dei ruoli;
+- Programmazione appuntamento con il Referente aziendale;
+- Retrospettiva sprint 4;
+- Presentazione degli dimostratori tecnologici realizzati;
+- Pianificazione sprint 5.
+
+== Rotazione dei ruoli
+
+Il gruppo attua la rotazione dei ruoli secondo la programmazione stabilita nel Piano di Progetto.
+
+== Meeting esterno con il Proponente
+
+È stato proposto un meeting al Proponente per il giorno 06/12/2023 alle ore 16:00. Attualmente il meeting è in attesa di conferma. Il meeting verterà sulle tecnologie, sui dimostratori tecnologici realizzati e sull'individuazione di requisiti non funzionali.
+
+== Dimostratori tecnologici
+
+Sono stati presentati tre dimostratori tecnologici, ciascuno focalizzato sulla comprensione di una specifica funzionalità:
+
++ *importazione SVG*: applicazione basata su _Three.js_ che genera un ambiente 3D a partire da un file SVG standard;
```

Comment:
l'ambiente generato è comunque 2D no? nel senso se ricordo bene dal poc di riccardo si vede solo l'immagine 2D inserita in un contesto 3D.